dj/construction/utils.py

```diff
@@ -106,3 +90,23 @@ def amenable_name(name: str) -> str:
             cont = []
```

Comment:
@shangyian I thought I had at one point seen an issue from you about the `_DOT_` in names. This is where that originates. Not sure if we have interest in revising that? I agree it looks pretty silly this might have been an overkill safety net

Reply:
@CircArgs I think my only issue was just that it looks a bit weird. But now that I've delved into it a bit more, I think it's fine to leave it, since I doubt most users are going to be actively looking at the SQL. It's more for us when debugging. We could think about renaming `_DOT_` to something less all caps, but not essential